concepticondata/conceptlists/Gauchat-1925-480.tsv

```diff
@@ -0,0 +1,481 @@
+Gauchat-1925-480-412	412	(in the) nose	(au) nez	nasu	138	1221	NOSE
+Gauchat-1925-480-413	413	(in the) teeth	aux dents	dĕntes	138	1380	TOOTH
+Gauchat-1925-480-414	414	(in the) neck	(au) cou	collu	138	1333	NECK
+Gauchat-1925-480-415	415	(in the) heart	(au) cœur	cŏr	138	1223	HEART
+Gauchat-1925-480-416	416	(in the) tigh	(à la) cuisse	cŏxa	138	800	THIGH
+Gauchat-1925-480-417	417	(in one) rib	(à une) côté	cŏsta	138	801	RIB
+Gauchat-1925-480-418	418	in one eye	à un œil	ŏclu	140		
+Gauchat-1925-480-419	419	in the eyes	aux yeux	ŏclos	140		
+Gauchat-1925-480-420	420	(in the) tongue	(à la) langue	lĭngua	140	1205	TONGUE
+Gauchat-1925-480-421	421	(in the) head	(à la) tête	tĕsta	140	1256	HEAD
+Gauchat-1925-480-422	422	(in one) nail	(à un) ongle	ŭngla	140	1258	FINGERNAIL
+Gauchat-1925-480-423	423	(in one) ear	(à une) oreille	aurĭcla, -ŭcla	140	1247	EAR
+Gauchat-1925-480-424	424	He fell	Il est tombé	cadūtu, -ēctu	142		
+Gauchat-1925-480-425	425	(she) fell	(elle est) tombée	cadūta, -ēcta	142		
+Gauchat-1925-480-426	426	on his/her buttocks	sur son derrière	cūlu	142	834	BUTTOCKS
+Gauchat-1925-480-427	427	He falls	Il tombe	cadit	142		
+Gauchat-1925-480-428	428	always	toujours	ad id ĭpsu	142	1676	ALWAYS
+Gauchat-1925-480-429	429	He has the arm	Il a le bras	brachiu	142		
+Gauchat-1925-480-430	430	naked	nu	nūdu	144	590	NAKED
+Gauchat-1925-480-431	431	the hand	la main	manu	144	1277	HAND
+Gauchat-1925-480-432	432	naked (feminine)	nue	nūda	144		
+Gauchat-1925-480-433	433	His/her body	Son corps	cŏrpus	144		BODY
+Gauchat-1925-480-434	434	is skinny	est maigre	macru	144		
+Gauchat-1925-480-435	435	The skin	La peau	pĕlle	144	763	SKIN
+Gauchat-1925-480-436	436	is thick	est épaisse	spĭssa	144		
+Gauchat-1925-480-437	437	The bone	L’os	ŏssu	146	1394	BONE
+Gauchat-1925-480-438	438	(is) thick	(est) épais	spĭssu	146	1244	THICK
+Gauchat-1925-480-439	439	He is right	Il est droit	dirēctu	146		
+Gauchat-1925-480-440	440	I kill	Je tue	tūto	146	1417	KILL
+Gauchat-1925-480-441	441	a fly	une mouche	mŭsca	146	1504	FLY (INSECT)
+Gauchat-1925-480-442	442	(a) spider	(une) araignée	aranea	146	843	SPIDER
+Gauchat-1925-480-443	443	an ant	une fourmi	formīcu, -a	148	587	ANT
+Gauchat-1925-480-444	444	(a) wasp	(une) guêpe	wĕspa	148	1517	WASP
+Gauchat-1925-480-445	445	(a) flea	(une) puce	pūlice	148	232	FLEA
+Gauchat-1925-480-446	446	(a) louse	(un) pou	pedŭclu	148	1392	LOUSE
+Gauchat-1925-480-447	447	The hunter	Le chasseur	captiatōre	148	1053	HUNTER
+Gauchat-1925-480-448	448	saw	a vu	vidūtu	148		
+Gauchat-1925-480-449	449	a hare	un lièvre	lĕpore	150	1190	HARE
+Gauchat-1925-480-450	450	(a) wolf	(un) loup	lŭpu	150	522	WOLF
+Gauchat-1925-480-451	451	(a) wild boar	(un) sanglier	singulare	150	3154	WILD BOAR
+Gauchat-1925-480-452	452	(an) eagle	(un) aigle	aquila	150	1905	EAGLE
+Gauchat-1925-480-453	453	a nice	un bel	bĕllu	150		
+Gauchat-1925-480-454	454	green bird	oiseau vert	aucĕllu vĭride	150		
+Gauchat-1925-480-455	455	The bird	L‘oiseau	aucĕllu	152	937	BIRD
+Gauchat-1925-480-456	456	makes	fait	facit	152		
```

Comment:
734?

Reply:
rather leave it unmapped. We tried to avoid to link to third singular.

Reply:
There are more such already mapped, though: arrive, c'est,  il ferme, il balaire...

Reply:
Then, I'd say, they should be unmapped.